ldb/models.py

```diff
@@ -182,9 +238,11 @@ def age(self):
             return today.year - born.year
 
     def save(self):
+        self._membership_status = self.membership_status
```

Comment:
Ik denk dat dit eigenlijk in de [clean](https://docs.djangoproject.com/en/1.8/ref/models/instances/#django.db.models.Model.clean) hoort.